components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -3319,7 +3319,8 @@
     public static final String ADD_CATEGORY_SQL = "INSERT INTO AM_API_CATEGORIES "
             + "(UUID, NAME, DESCRIPTION, TENANT_ID) VALUES (?,?,?,?)";
 
-    public static final String GET_CATEGORIES_BY_TENANT_ID_SQL = "SELECT * FROM AM_API_CATEGORIES WHERE TENANT_ID = ?";
+    public static final String GET_CATEGORIES_BY_TENANT_ID_SQL = "SELECT * FROM AM_API_CATEGORIES WHERE TENANT_ID = ? "
```

Comment:
Its more efficient to provide the exact columns that are required by the query as opposed to using the `*` notation in the SELECT statement

Reply:
Fixed with 6f5136d